### DISCUSSION TITLE
Los提示“被禁止从公网访问”

### DISCUSSION THREAD

**Discussion by dmjtian** [jeessy2/ddns-go#1001] [Q&A] [answered]
设备：群晖Nas
- [✅] 局域网IP访问
-  [✅] 局域网域名访问
-  [❌] 公网域名访问
- 以下为Logs提示信息：
> 2024/01/31 00:25:08 You can use 'sudo ./ddns-go -s install' to install service
2024/01/31 00:25:08 监听 :9876
2024/01/31 00:25:10 你的IP 223.xxx.xxx.104 没有变化, 域名 nas.xxxxxx.cn
2024/01/31 00:25:10 你的IP 2409:xxxx:xxxx:xxxx::38 没有变化, 域名 nas.xxxxxx.cn
2024/01/31 00:26:21 "Remote: [2409:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx]:58239" 被禁止从公网访问
2024/01/31 00:26:21 "Remote: [2409:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx]:58239" 被禁止从公网访问
2024/01/31 00:26:36 "Remote: 192.168.31.181:58258" 请求登陆
2024/01/31 00:26:36 "Remote: 192.168.31.181:58258" 请求登陆
2024/01/31 00:26:36 "Remote: 192.168.31.181:58259" 请求登陆
2024/01/31 00:26:44 "Remote: 192.168.31.181:58259" 请求登陆
2024/01/31 00:26:44 "Remote: 192.168.31.181:58259" 请求登陆
2024/01/31 00:27:26 "Remote: [2409:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx]:58320" 被禁止从公网访问
2024/01/31 00:35:11 IPv4未改变, 将等待 5 次后与DNS服务商进行比对
2024/01/31 00:35:11 IPv6未改变, 将等待 5 次后与DNS服务商进行比对

请求大家帮助！谢谢！

**Comment by dmjtian** [comment]
补充：

- [❌ ] 公网Ping域名
- [✅ ] 内网Ping域名

**Comment by jeessy2** [comment, marked as answer]
局域网IP进入，把禁止公网访问取消勾选
